.github/workflows/main.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        tox_env: ["py36", "py37", "py38", "linting", "docs"]
```

Comment:
You removed "docs" from tox_env matrix but didn't remove the "testenv:docs" from tox.ini file.
Is that intentional?

Reply:
We should add a notification e-mail in readthedocs to better notice failures in building the docs.
https://readthedocs.org/dashboard/conda-devenv/notifications/

Reply:
> You removed "docs" from tox_env matrix but didn't remove the "testenv:docs" from tox.ini file.
Is that intentional?

Yes, wanted to leave a way to easily build the docs locally.

> We should add a notification e-mail in readthedocs to better notice failures in building the docs.

You think that's necessary? Note that now we have a dedicated ReadTheDocs build:

![image](https://user-images.githubusercontent.com/1085180/151852123-638c1a02-b4cf-40ea-8b4e-9be074e2484c.png)

If you prefer to have the email notification: which email should we use?

Reply:
I did ask myself that =)
I will be nice if git hub have one that just forwards to the maintainers but ask far I can tell there is not one.
Well I will add my.

Reply:
OK 👍